refactor: remove unused dependencies and code

mitagg opened the pull request.

Commit:
refactor: remove unused dependencies and code

Commit:
refactor: use object shorthand notation

kirchoni reviewed (approved):
Linked the package locally and every command seems to work as expected.
Do we need the `e2e-utils.js` file, couldn't find it's usage neither in kendo-react-tasks, nor in any of our packages, and everything seems to work without it?

mitagg commented:
Actually, e2e-utils code should be part of test-utils, as for the usage we are currently not using it, but it could be used [here](https://github.com/telerik/kendo-react-private/blob/214d3bf44f17513b059663d881d54b534b5226b1/packages/animation/test/test.tsx#L217) and [here](https://github.com/telerik/kendo-react-private/blob/92bf94c5ea36ac449ef8fc9c4470d15f6d951aa7/packages/popup/test/test.tsx#L33)

Commit:
refactor: move withRoot in test utils

Move code withRoot method from e2e utils to test one, as it is used there.
Leave e2e-utils as placeholder for future e2e utils.

Commit:
chore: add editorconfig
